docs/guides/color.md

```diff
@@ -0,0 +1,290 @@
+(color)=
+
+# Colors and colormaps
+
+This document describes how to programmatically define colors and colormaps in napari.
+It is intended for users of napari that are comfortable using the napari API and
+developers who want to include color as part of something they are building.
+
+
+## Colors
+
+In many places, napari allows the specification of color.
+This might be the color of an individual point, the color of text annotations, or a color value in a colormap.
+For example, the face color of all points in a layer can be defined as red.
+
+```python
+In [1]: points.face_color = 'red'
+```
+
+### Standard form
+
+Internally, napari typically coerces an input color value like `'red'` to an RGBA (red, green, blue, alpha)
+array with `shape=(4,)` and `dtype=np.float32` where the values are in `[0, 1]`.
+When storing N color values, a similar array of `shape=(N, 4)` is used where each row represents a single color value.
+For example, reading the points layer's red face color gives us such an array.
+
+```python
+In [2]: points.face_color
```

Comment:
You should describe that there are three points in this example.

Reply:
Done.